e2e/tests/admin/sidebar/navigation.test.ts

```diff
@@ -0,0 +1,161 @@
+import {SidebarPage} from '@/admin-pages';
+import {expect, test} from '@/helpers/playwright/fixture';
+
+test.describe('Ghost Admin - Sidebar Navigation', () => {
+    const mainNavItems = [
+        {name: 'Analytics', expectedPath: '/ghost/#/analytics'},
+        {name: 'View site', expectedPath: '/ghost/#/site'},
+        {name: 'Posts', expectedPath: '/ghost/#/posts'},
+        {name: 'Pages', expectedPath: '/ghost/#/pages'},
+        {name: 'Tags', expectedPath: '/ghost/#/tags'},
+        {name: 'Members', expectedPath: '/ghost/#/members'}
+    ];
+
+    test.describe('main navigation', () => {
+        mainNavItems.forEach(({name, expectedPath}) => {
+            test(`clicking ${name} - navigates and shows active state`, async ({page}) => {
+                const sidebar = new SidebarPage(page);
+
+                await sidebar.goto('/ghost');
+
+                await sidebar.clickNavItem(name);
+
+                await expect(page).toHaveURL(new RegExp(expectedPath.replace(/[#]/g, '\\$&')));
```

Comment:
It would be nice if we can clean up this regex and not do a bunch of string manipulation. Tweaking the expected paths above would allow us to do so.